web-admin/src/features/billing/plans/PlanQuotas.svelte

```diff
@@ -23,7 +23,7 @@
       : "Unlimited";
 
   $: usageMetrics = getOrganizationUsageMetrics(organization);
-  $: total = $usageMetrics?.data?.reduce((s, m) => s + m.size, 0) ?? 0;
+  $: totalOrgSize = $usageMetrics?.data?.reduce((s, m) => s + m.size, 0) ?? 0;
```

Comment:
Nit: I read `totalOrgSize` as "number of people in the organization". `totalOrgUsage`?